src/Core/src/App.Metrics.Formatters.Ascii/MetricsTextOutputFormatter.cs

```diff
@@ -59,23 +59,15 @@ public MetricsTextOutputFormatter(MetricsTextOptions options, MetricFields metri
 
             var serializer = new MetricSnapshotSerializer();
 
-            using (var streamWriter = new StreamWriter(output, _options.Encoding, bufferSize: 1024, leaveOpen: true))
-            {
-                using (var textWriter = new MetricSnapshotTextWriter(
-                    streamWriter,
-                    _options.Separator,
-                    _options.Padding,
-                    _options.MetricNameFormatter))
-                {
-                    serializer.Serialize(textWriter, metricsData, MetricFields);
-                }
-            }
-
-#if NETSTANDARD1_6
-            return Task.CompletedTask;
-#else
-            return AppMetricsTaskHelper.CompletedTask();
-#endif
+            using var streamWriter = new StreamWriter(output, _options.Encoding, bufferSize: 1024, leaveOpen: true);
```

Comment:
this code should be replaced by:

`#if NETSTANDARD2_1
            await using var streamWriter = new StreamWriter(output, _options.Encoding, bufferSize: 1024, leaveOpen: true);
#else
            using var streamWriter = new StreamWriter(output, _options.Encoding, bufferSize: 1024, leaveOpen: true);
#endif`